rust/operator-binary/Cargo.lock

```diff
@@ -0,0 +1,1991 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I don't think we need .lock files in the individual crates, the top level one for the workspace Cargo.toml should contain everything, no?